zeppelin-zengine/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -411,6 +420,8 @@ public String getConfDir() {
     ZEPPELIN_INTERPRETER_CONNECT_TIMEOUT("zeppelin.interpreter.connect.timeout", 30000),
     ZEPPELIN_ENCODING("zeppelin.encoding", "UTF-8"),
     ZEPPELIN_NOTEBOOK_DIR("zeppelin.notebook.dir", "notebook"),
+    ZEPPELIN_NOTEBOOK_S3_BUCKET("zeppelin.bucket.name", "zeppelin"),
+    ZEPPELIN_NOTEBOOK_S3_USER("zeppelin.bucket.user", "user"),
```

Comment:
Thanks for the updating the name. but can we have little bit of consistency between name of env variable and name of property? to me "zeppelin.notebook.s3.user" is more consistent. Is there special reason you chose this name?